VERSION.south.sinusoid

```diff
@@ -1,2 +1,2 @@
-foglamp_south_sinusoid_version=1.0.0
-foglamp_version>=1.3
\ No newline at end of file
+foglamp_south_sinusoid_version=2.0.0
+foglamp_version>=1.5
```

Comment:
kindly change to 1.4 (as it will go to develop only), we will fix it. when FORMAL release will happen.